frontend/src/components/orgFacilities/ManageFacilities.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <v-container class="pa-0 text-body-1" fluid>
+    <v-row no-gutters>
+      <v-col>
+        <p>View and update your facilities.</p>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-col>
+        <h2>Facility Details</h2>
+        <p class="mt-2">
+          <v-icon>mdi-information</v-icon>
+          Note: The information marked with * will be shared on Child Care Map. Please do not include any personal
+          information.
+        </p>
+      </v-col>
+    </v-row>
+    <template v-if="facilitiesLoading">
+      <v-row v-for="n in skeletons" :key="n" no-gutters>
+        <v-col class="mt-2">
+          <v-card variant="outlined" class="soft-outline fill-height px-2">
+            <v-skeleton-loader type="article" />
+          </v-card>
+        </v-col>
+      </v-row>
+    </template>
+    <template v-else>
+      <v-row>
+        <v-col>
+          <h3>Active Facilities</h3>
+        </v-col>
+      </v-row>
+      <FacilityList :facilities="activeFacilities" />
+      <v-row v-if="inactiveFacilities">
+        <v-col>
+          <h3>Inactive Facilities</h3>
+        </v-col>
+      </v-row>
+      <FacilityList :facilities="inactiveFacilities" />
+    </template>
+  </v-container>
+</template>
+<script>
+import { mapActions, mapState } from 'pinia';
+import { isEmpty } from 'lodash';
+
+import { useApplicationStore } from '@/store/application';
+import { useOrganizationStore } from '@/store/ccof/organization';
+import { ORGANIZATION_FACILITY_STATUS_CODES } from '@/utils/constants.js';
+
+import alertMixin from '@/mixins/alertMixin.js';
+import FacilityList from '@/components/orgFacilities/FacilityList.vue';
+
+export default {
+  name: 'ManageFacilities',
+  components: {
+    FacilityList,
+  },
+  mixins: [alertMixin],
+  data() {
+    return {
+      facilitiesLoading: false,
+      activeFacilities: [],
+      inactiveFacilities: [],
+    };
+  },
+  computed: {
+    ...mapState(useOrganizationStore, ['organizationId', 'facilities', 'loadedModel']),
+    ...mapState(useApplicationStore, ['programYearId']),
+    skeletons() {
+      if (this.loadedModel.numberOfFacilities > 10) {
+        return 10;
+      }
+      return this.loadedModel.numberOfFacilities;
+    },
+  },
+  async mounted() {
+    try {
+      if (isEmpty(this.facilities)) {
+        this.facilitiesLoading = true;
+        await this.loadFacilities(this.organizationId);
```

Comment:
As discussed, I think facilities and loadFacilities can exist in the component. I don't think there is a good case for keeping them in the store at this point.